.github/workflows/mypy.yml

```diff
@@ -0,0 +1,19 @@
+name: mypy
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies and mypy
+      run: |
+        pip install cython mypy
+        pip install -r requirements.txt
+    - name: Verify mypy types
+      run: |
+        mypy --version
+        mypy --ignore-missing-imports . 
```

Comment:
I think at some point we'll have a `mypy.ini` file that we can use to customize what options we run with - that is possibly more sustainable?

Reply:
Sounds good to me.